imports/users-presence/server/activeUsers.js

```diff
@@ -0,0 +1,20 @@
+import { UserPresenceEvents } from 'meteor/konecty:user-presence';
+
+import { Notifications } from '../../../app/notifications/server';
+
+UserPresenceEvents.on('setUserStatus', (user, status/* , statusConnection*/) => {
+	const {
+		_id,
+		username,
+		// name,
+		// utcOffset,
+	} = user;
+
+	// since this callback can be called by only one instance in the cluster
+	// we need to brodcast the change to all instances
+	Notifications.notifyLogged('user-status', {
```

Comment:
what do you think about to use an array, to save some bytes (as we always send the same payload, we know the position of each property). AND I would love use 0,1,2,3 instead of 'offline', 'online', 'away' and 'busy', just because the streamer uses more keys than the original DDP so we could make few tweaks to reduce the difference <3

Reply:
done!